addons/payment_worldline/models/payment_transaction.py

```diff
@@ -113,15 +113,15 @@ def _worldline_create_checkout_session(self):
                 },
                 'customer': {  # required to create a token and for some redirected payment methods
```

Comment:
`lang` is not required either.

Reply:
Would it be possible for us not to have one? :thinking: